pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -1249,19 +1232,42 @@ public void updateKafkaPartitionsIfNecessary(String realtimeTableName, TableConf
      }
    }
    */
-  private ZNRecord generatePartitionAssignment(String topicName, int nPartitions, List<String> instanceNames,
-      int nReplicas) {
+  private ZNRecord generatePartitionAssignment(TableConfig tableConfig, String topicName, int nPartitions, List<String> instanceNames,
+      int nReplicasNU) {
```

Comment:
What is the difference between nReplicasNU parameter and nReplicas on line 1237? They look like they're the same? Do we need both?

Reply:
No, we can remove the NU. The NU was meant for not used. Let me take a look